apiary.apib

```diff
@@ -8673,6 +8673,63 @@ idComponent and configurationId.
                 }
             ]
 
+## Global search [/v2/storage/global-search?limit={limit}&offset={offset}&types={types}&projectIds={projectIds}]
```

Comment:
tu vlastne este chyba `query`

Reply:
Mozno by som tam rovno pridal aj tie dva nove parametre https://github.com/keboola/connection/blob/4bb6156d82128cead6fcfbf736f7a8fc807beb70/src/Storage/GlobalSearch/GlobalSearchRequest.php#L19-L20 uz ked to budes upravovat, na testy spravim task

Reply:
jo, dobrý nápad, je to tam

A že nám chybí ta "query" je dobrá haluz :-D